test/bench/ok/heap-32.drun-run.ok

```diff
@@ -1,5 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-debug.print: (50_227, +91_377_860, 1_433_012_379)
-debug.print: (50_070, +102_586_000, 1_504_942_528)
+debug.print: (50_227, +30_261_252, 666_216_815)
```

Comment:
@crusso obviously these number came down, should we increase the iteration counts?

Reply:
I dunno, then you'd hide the improvement. Why not leave as is?

Reply:
I am fine with that, if your original intention wasn't to fill up the heap or similar...